ga4gh/serverconfig.py

```diff
@@ -26,6 +26,12 @@ class BaseConfig(object):
     SIMULATED_BACKEND_VARIANT_DENSITY = 0.5
     SIMULATED_BACKEND_NUM_VARIANT_SETS = 1
 
+    # Options for OIDC
+    # OIDC_PROVIDER = "https://accounts.google.com"
+    # OIDC_CLIENT_ID = "XXXX"
+    # OIDC_CLIENT_SECRET = "XXXX"
+    SERVER_NAME = "localhost:8080"
```

Comment:
It's hard to set automatically because the host could have several names, so flask doesn't know which one to use. Best to be explicit, and localhost is fine for dev.